src/content/docs/en/guides/cms/builderio.mdx

```diff
@@ -3,19 +3,507 @@ title: Builder.io & Astro
 description: Add content to your Astro project using Builder.io’s visual CMS
 type: cms
 service: Builder.io
-stub: true
+stub: false
 i18nReady: false
 ---
 import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+import FileTree from '~/components/FileTree.astro';
 
+[Builder.io](https://www.builder.io/) is a visual CMS that supports drag and drop content editing for building web sites.
 
+At the end of this recipe you'll be able to create blog posts in the Builder app and have your Astro site update with this new content automatically using a webhook.
+
+To do this, you'll use Builder's [Content API](https://www.builder.io/c/docs/content-api) and [Qwik API](https://www.builder.io/c/docs/qwik-api) to connect your Builder space to Astro with zero client-side JavaScript.
+
+:::tip[Official resource]
```

Comment:
No, I didn't make any changes here. I just pointed out that this tip isn't relevent to the reader at *this* point of the instrutions. They don't need to know about what happens when you're making fetch calls until they are making fetch calls. I think this is worthy of a tip, but only once we get to the point where the reader is doing this.

(And, since I got cut off from editing because of my flight, I just haven't made it to that part of the document yet! So I didn't delete from here, or move anywhere else yet.)

Reply:
I see now, let's do this! thanks for the explanation.